user-guide/manual.adoc

```diff
@@ -392,6 +392,12 @@ The bundle has now completed the required BSL interactions, and the bundle conte
 .Visual Representation of Per-Bundle Workflow
 plantuml::static/bundle-workflow.puml[]
 
+The BSL Frontend API consists of two primary functions for per-bundle operation: 
```

Comment:
Small distinction, but the "frontend" is a library and the "BSL service API" is what I think this is about. The thing that the BPA uses to call into the BSL. I may need to create a more detailed and accurate interaction diagram.

Reply:
I'm making more edits to try to clarify this, I think the language can stay with the "service API" being a subset of "frontend API" called by the BPA.